### ISSUE TITLE
Request standardization for terraform destroy and apply arguments

### ISSUE THREAD

**Issue by ScriptMyJob** [hashicorp/terraform#17385]
### Terraform Version
```
14:06 MacOS: Terraform/ $ terraform --version
Terraform v0.11.3
+ provider.aws v1.6.0
+ provider.null v1.0.0

14:06 MacOS: Terraform/ $
```

### Terraform Configuration Files
This is not applicable

### Debug Output
This is not applicable

### Crash Output
This is not applicable

### Expected Behavior
It would be nice if `-y` was aliased for `-force` for `terraform destroy` and `-auto-approve` for `terraform apply`

### Actual Behavior
```
14:05 MacOS: Terraform/ $ terraform apply -y
flag provided but not defined: -y
Usage: terraform apply [options] [DIR-OR-PLAN]

  Builds or changes infrastructure according to Terraform configuration
  files in DIR.

  By default, apply scans the current directory for the configuration
  and applies the changes appropriately. However, a path to another
  configuration or an execution plan can be provided. Execution plans can be
  used to only execute a pre-determined set of actions.

Options:
... (removed for readability)
14:06 MacOS: Terraform/ $ 
```

### Steps to Reproduce
`terraform apply -y`
or 
`terraform destroy -y`

### Additional Context
I think that this would help developers in Staging and Development environments that can be stood up and torn down multiple times.

**Comment by ScriptMyJob** [comment]
This is not so much an issue as it is a feature request.  Sorry if I put this in the wrong place.

**Comment by apparentlymart** [comment]
Hi @ScriptMyJob! Thanks for this feature request.

Over in #17218 there is a change to make `-auto-approve` work on both `terraform apply` and `terraform destroy`.

We intentionally chose a very explicit name for this option because it's a dangerous option that should generally be used only in automation scenarios (where length of typing is not an issue). We don't want to have a single-character flag for something this dangerous.

The recommended workflow for _interactive_ use is to type `yes` at the interactive prompt. I understand that this can be inconvenient if you are very rapidly creating an destroying a low-value environment, but Terraform's primary workflow designed with the common case of a high-value environment in mind.

If you have a codebase that has a workflow of frequent creating and destroy, I'd suggest optimizing that specific workflow with a codebase-specific wrapper script (or Makefile, etc) that includes the `-auto-approve` option. By including it in the relevant codebase you'd ensure that it can't be accidentally applied to a higher-value environment.

The current inconsistency between `-force` and `-auto-approve` will be addressed once #17218 is merged. I hope my rationale here for not implementing `-y` makes sense. Since we already have that PR open, I'm going to close this just to consolidate. Thanks again for opening this.
